drivers/lcd/ssd1306_base.c

```diff
@@ -782,7 +783,7 @@ static int ssd1306_setpower(FAR struct lcd_dev_s *dev, int power)
         {
           /* Configure display and turn the display on */
 
-          ret = ssd1306_configuredisplay(priv);
+          ret = ssd1306_configuredisplay(dev);
```

Comment:
why the delay configuration here does work?

Reply:
Hi @xiaoxiang781216,

In our case, this SSD1306 display is pulling power from the I2C lines because the supply voltage of the display is not active yet.
This causes the I2C transactions to look OK, but in fact it is not being configured correctly.

I've checked, but there is no way to read out this display if it went OK.

We thought of a better solution and opened an other PR, see closing comment.

Reply:
If so, look like you have to turn on the display power before i2c transaction and turn off the display power after i2c. Of course, you need some form of reference counting, so the display power doesn't turn off prematurely if it's really used.